core/services/validators.go

```diff
@@ -49,7 +49,11 @@ func ValidateBridgeType(bt *models.BridgeTypeRequest, store *store.Store) error
 	if _, err := models.NewTaskType(bt.Name.String()); err != nil {
 		fe.Merge(err)
 	}
-	if _, err := url.Parse(bt.URL.String()); err != nil {
+	u := bt.URL.String()
+	if len(strings.TrimSpace(u)) == 0 {
+		fe.Add("URL must be present")
+	}
+	if _, err := url.Parse(u); err != nil {
```

Comment:
Looks like you already have a url here and don't need to parse it again?

Reply:
This is used to add a JSON-API error

Reply:
Oh sure, I just think this code path looks impossible to reach.